### PR TITLE
[SETL-13] Add visualization to etl processes for better understanding

### PR DESCRIPTION
**Pull request by kubudi**

### CODE REVIEW

**Comment by furkanvarol** [comment]
It looks nice! We can merge it after #6.

File: README.md [hunk 1]
```diff
@@ -8,30 +8,40 @@
 
 spark-etl is a Scala-based project and it is developing with Spark. So it is scalable and distributed. spark-etl will process data from N source to N database.
 The project structure:
-- Extract
+
+**Extract**
+
+<img src="https://media.giphy.com/media/26BRIuiQfAxf5cENO/giphy.gif" alt="alt text" width="250" height="250">
```
Comment (msimav) [inline README.md:14]: Github markdown has [images](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images)

**Comment by hsnthn** [comment]
@kubudi I have created an issue for this. Can you please update commit message?
https://spark-etl.atlassian.net/browse/SETL-13

File: README.md (hunk 1 above)
Comment (furkanvarol) [inline README.md:14]: example;
`extract` =>
extract

Comment (kubudi) [inline README.md:14]: alt text

Yeah but It doesn't support external links

Comment (kubudi) [inline README.md:14]: Ah, sorry. You just cannot resize images.
There is a notation like `extract`
but I couldn't managed to make it work

**Pull request merged**